### ISSUE TITLE
MessageDigest not available

### ISSUE THREAD

**Issue by aalku** [boneskull/digistump-board-index#1]
Hello
I try to use Digispark with Arduino 1.6.4 but when trying to install the board I get the error in the image.

![image](https://cloud.githubusercontent.com/assets/12587056/7789048/54870f6a-0253-11e5-8467-b143804fe13a.png)

I have configured this url for additional boards in arduino preferences:
  http://boneskull.com/digistump-board-index/package_digistump_index.json

As you can see.
![image](https://cloud.githubusercontent.com/assets/12587056/7789068/c28cdc56-0253-11e5-85e6-e4c7fa6bb034.png)

Is this a bug or something? Thank you.

**Comment by boneskull** [comment]
yeah.  this package isn't working properly yet.  I posted on the digispark forums for help

**Comment by aalku** [comment]
I think I found it. The checksum algorithm is missing in the last json part.

![image](https://cloud.githubusercontent.com/assets/12587056/7800633/9b887f1e-0318-11e5-95c8-f4a173aec667.png)

**Comment by pierrotm777** [comment]
Always this issue with the json file

**Comment by tschertel** [comment]
Could this be fixed, please?

**Comment by tschertel** [comment]
I was able to install, but now it shows a different error message:

    Warning: platform.txt from core 'Digistump AVR Boards' contains deprecated recipe.ar.pattern="{compiler.path}{compiler.ar.cmd}" {compiler.ar.flags} "{build.path}/{archive_file}" "{object_file}", automatically converted to recipe.ar.pattern="{compiler.path}{compiler.ar.cmd}" {compiler.ar.flags} "{archive_file_path}" "{object_file}". Consider upgrading this core.
    Warning: platform.txt from core 'Digistump AVR Boards' contains deprecated recipe.c.combine.pattern="{compiler.path}{compiler.c.elf.cmd}" {compiler.c.elf.flags} -mmcu={build.mcu} -o "{build.path}/{build.project_name}.elf" {object_files} "{build.path}/{archive_file}" "-L{build.path}" -lm, automatically converted to recipe.c.combine.pattern="{compiler.path}{compiler.c.elf.cmd}" {compiler.c.elf.flags} -mmcu={build.mcu} -o "{build.path}/{build.project_name}.elf" {object_files} "{archive_file_path}" "-L{build.path}" -lm. Consider upgrading this core.
    Warning: platform.txt from core 'Digistump AVR Boards' contains deprecated recipe.ar.pattern="{compiler.path}{compiler.ar.cmd}" {compiler.ar.flags} "{build.path}/{archive_file}" "{object_file}", automatically converted to recipe.ar.pattern="{compiler.path}{compiler.ar.cmd}" {compiler.ar.flags} "{archive_file_path}" "{object_file}". Consider upgrading this core.
    Warning: platform.txt from core 'Digistump AVR Boards' contains deprecated recipe.c.combine.pattern="{compiler.path}{compiler.c.elf.cmd}" {compiler.c.elf.flags} -mmcu={build.mcu} -o "{build.path}/{build.project_name}.elf" {object_files} "{build.path}/{archive_file}" "-L{build.path}" -lm, automatically converted to recipe.c.combine.pattern="{compiler.path}{compiler.c.elf.cmd}" {compiler.c.elf.flags} -mmcu={build.mcu} -o "{build.path}/{build.project_name}.elf" {object_files} "{archive_file_path}" "-L{build.path}" -lm. Consider upgrading this core.
    Invalid library found in C:\Users\tomasschertel\AppData\Local\Arduino15\packages\digistump\hardware\avr\1.5.4\libraries\Digispark_Examples: C:\Users\tomasschertel\AppData\Local\Arduino15\packages\digistump\hardware\avr\1.5.4\libraries\Digispark_Examples
    Invalid library found in C:\Users\tomasschertel\AppData\Local\Arduino15\packages\digistump\hardware\avr\1.5.4\libraries\Digispark_Examples: C:\Users\tomasschertel\AppData\Local\Arduino15\packages\digistump\hardware\avr\1.5.4\libraries\Digispark_Examples`

**Comment by pierrotm777** [comment]
Return always an error !
